### DISCUSSION TITLE
Allow createResource disable suspense integration.

### DISCUSSION THREAD

**Discussion by minht11** [solidjs/solid#1165] [Ideas]
This is feature request, I want `createResource` have an option which disables integration with suspense, making it regular async fetching utility.

My use case:
I have toplevel `Suspense` boundary wrapping router pages.
Using route data function I fetch all of the list ids' from indexeddb async using `createResource`.
Now inside actual page there is virtual list taking that flat id list, each list item then fetches their own data as they need.

As it is right now I can't use `createResource` inside list items, because after initial load and because virtual list recycles nodes, fetching new data would retrigger top level page load indicators, flashing if not used with `startTransition`, I tried using `SuspenseList`, but because it relies on element ordering, that also has issues like click handlers not working if list is scrolled. Overall it wouldn't even be great fit to use suspense inside each list item even if I wasn't facing issues like that.

`createResource` already has great functionality for async data loading, so it would be nice if it could also be repurposed as generic data loading tool, not only when using suspense.

**Comment by ryansolid** [comment]
Hmm.. I think a primitive could be built here pretty easily. However, `createResource` is specifically created to be minimal interface for Suspense. Like to make a simple primitive to not use Suspense would be 1/6th the code. This is pretty trivial in user space. Mostly I can't guarantee it will always align with the non-suspense case.
